documentation/dryad_install.md

```diff
@@ -49,16 +55,12 @@ You should end up with a directory structure that looks like this one.
     └── stash_engine
 ```
 
-Your config files are currently in a seperate directory from your application. It can be handy to keep them apart from the application so that you can back them up or commit them to a private repository for configuration separate from the application.  The application will need to have these configuration files symlinked into the application. to symlink the files in using a bash shell, type these commands:
-
-```
-cd dryad
-mkdir config/tenants
-./symlink_config.sh
-```
-
 After you add or remove any configuration file, such as a yml file in the tenants directory, you will want to run the script above again update symlinks to config in the application.
```

Comment:
This line can be taken out entirely, since the tenants directory itself is now a symlink.